Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY . .
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app .
+
+RUN pip install -r requirements.txt && \
+    python manage.py migrate
+
+EXPOSE 8080
+
+ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/d346451c-f054-4aad-a160-8d53eb556bf0)